macos-py3.10-requirements.txt

```diff
@@ -1,32 +1,33 @@
+WARNING: --strip-extras is becoming the default in version 8.0.0. To silence this warning, either use --strip-extras to opt into the new default or use --no-strip-extras to retain the existing behavior.
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
nit: I try to generate this with python 3.10 just to make sure it works there. on macos i do:

```
brew install python@3.10
python3.10 -m venv venv
...
...
pip-compile --all-extras pyproject.toml
```

Reply:
fixed.